src/Directory.Build.targets

```diff
@@ -24,7 +23,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IncludeRx)' == 'true'">
-    <PackageReference Include="System.Reactive.Linq" Version="$(RxVersion)" />
+    <PackageReference Include="RxFree" Version="$(RxVersion)" Pack="false" />
```

Comment:
Does this mean we would have 2 RX implementations in the IDE now?

Reply:
Far from it. `RxFree` has literally 4 types, ultra-basic-barebones that are the absolute bare minimum that anyone producing observables would need. RX itself has a *ton* of stuff for *consumers* of said observables, but producing them is actually quite easy and incurring the whole RX cost for a couple classes that are really little more than helpers (the ["most" complex one is <300](https://github.com/kzu/RxFree/blob/master/RxFree/System/Subject.cs) including quite a few comments) is not worth it. Especially since we need to keep bumps mostly in sync (had issues running unit and integration tests otherwise).

Reply:
TBH, I could have copy/pasted `Disposable/CompositeDisposable`, `Subject<T>` and `ObservableExtensions` right into this project, but since I'm using those same 4 types in other places, it's just easier to just reference the source-only package I have for them already ;)